floyd/cli/data_upload_utils.py

```diff
@@ -56,17 +60,22 @@ def initialize_new_upload(data_config, access_token):
 
     creds = DataClient().new_tus_credentials(data_id)
     if not creds:
+        # TODO: clean up
         sys.exit(1)
 
-    data_endpoint = TusDataClient().initialize_upload(tarball_path,
-                                                      metadata={"filename": data_id},
-                                                      auth=creds)
+    data_resource_id = creds[0]
```

Comment:
Yes, so the data_id here is the unique identifier for the data module itself, which is a wrapper of data_resource(the actual file) and a bunch of metadatas.`data_resource_id` is the unique identifier we use to store the data on disk.